.github/workflows/notes.md

```diff
@@ -27,32 +27,56 @@ Note: On Windows "Server" you may need to [install vcruntime140.dll](https://lea
 3. Extract: `tar xzvf nvim-macos-x86_64.tar.gz`
 4. Run `./nvim-macos-x86_64/bin/nvim`
 
-### macOS (arm64)
+### macOS (aarch64)
```

Comment:
Why rename this? The new linux arm build can be called `arm64`. Then they'll be consistent. And it's more discoverable for users. And easier for scripts that download our releases (used very commonly in CI).

Reply:
Continuing https://github.com/neovim/neovim/pull/32194#discussion_r1929566813 : 

> > scripts can use `$ARCH` to download the right appimage if we stick to the standard.)
> 
> That's a good reason, if scripts are doing that. Do you have an example? The example I've seen is our own node-client which points to various architectures, and it sure looks silly that "x86" and "arm" are spelled different ways for different OSes: https://github.com/neovim/node-client/blob/aa8fff0b7e486caf66a95befeed5805ae7e458f1/.github/workflows/ci.yml#L33-L43
> 
> Any scripts that download all the release archives for whatever reason have to treat each name specially, they can't "calculate" the name using simple logic.

Reply:
There's no good reason for renaming the macOS artifacts (which will break downstream scripts).